supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/labels/Label.scala

```diff
@@ -0,0 +1,59 @@
+package cromwell.backend.impl.jes.labels
+
+/**
+  * Construct via Label.safeLabel
+  */
+case class Label private[labels](key: String, value: String) {
+
+  def +(that: Label): Labels = Labels(Vector(this, that))
+}
+
+object Label {
+
+  def validate(s: String) = "[a-z]([-a-z0-9]*[a-z0-9])?".r.pattern.matcher(s).matches && s.length <= 63
+
+  /**
+    * Enforce the constraint:
+    *  - Must fix the regex [a-z][-a-z0-9]*[a-z0-9]
+    *  - Must be shorter than 63 characters total
```

Comment:
Also I'd make a constant from the 64 both because I don't know what it stands for and to avoid constant littering.